content/en/tutorials/entity-picking.md

```diff
@@ -41,15 +45,36 @@ PickerRaycast.prototype.onSelect = function (e) {
 
 ## Frame Buffer Picking
 
-Frame buffer based picking uses the [pc.Picker][3] class to render the scene to a internal buffer. When the mouse is clicked the color of the buffer at the mouse location is used to determine which mesh instance was selected. This has some advantages and disadvantages over the collision based method. Advantages include: using a rectangle to pick many items in a scene at once, this method doesn't have require the physics library to be included and active in your project. The main disadvantage is that this uses the `readPixels` method which stalls the graphics pipeline. This can have serious rendering performance implications particularly on mobile.
+Frame buffer based picking uses the [pc.Picker][3] class to render the scene to a internal buffer. When the mouse is clicked the color of the buffer at the mouse location is used to determine which mesh instance was selected. This has some advantages and disadvantages over the collision based method.
+
+Advantages include:
+
+* Able to use a rectangle to pick many items in a scene at once
+* Doesn't require the physics library to be included which reduces preload time.
+
+The main disadvantage is that this uses the `readPixels` method which stalls the graphics pipeline. This can have serious rendering performance implications particularly on mobile.
```

Comment:
I would add that the performance impact is higher on GPU heavy applications, and lower or GPU light applications.

Reply:
```suggestion
The main disadvantage is that this uses the `readPixels` method which stalls the graphics pipeline. This can have serious rendering performance implications particularly on mobile and GPU heavy applications. 
```

Reply:
Is it possible to only render a layer/layers to the picker?

Reply:
good suggestion!

and yes to your question, I added optional array of layers recently:
https://developer.playcanvas.com/en/api/pc.Picker.html#prepare

Reply:
I shall extend the example! :)

Reply:
Updated the example to allow for an layer array to pick from